### PR TITLE
Redeploy documentation

### PR DESCRIPTION
**Pull request by coffee-cup**
This PR
- Added a banner to indicate if a feature is for priority boarding users only
- Adds documentation for the (priority boarding) redeploy feature
- Rearanged some of the deployment docs

### CODE REVIEW

File: src/components/Banner.tsx [hunk 1]
```diff
@@ -63,4 +70,14 @@ export const Banner: React.FC<Props> = ({ children, hideIcon, ...props }) => {
...
   );
-};
\ No newline at end of file
+};
+
+export const PriorityBoardingBanner: React.FC = () => {
+  return (
+    <Banner variant="primary" icon={Star}>
+      This feature is only available to{" "}
+      <Link href="/reference/priority-boarding">priority boarding</Link>{" "}
```
Comment (ndneighbor) [inline src/components/Banner.tsx:79]: Nit: should be capital case. `Priority Boarding`

File: src/pages/deploy/deployments.md [hunk 2]
```diff
@@ -104,29 +118,30 @@ width={1518} height={502} quality={80} />
...
+an new deployment with the exact same code and build/deploy configuration.
 
 <Image
-src="https://res.cloudinary.com/railway/image/upload/v1645149733/docs/logs_seqcps.png"
-alt="Animation of Log Filtering"
+src="https://res.cloudinary.com/railway/image/upload/v1666380373/docs/redeploy_ghinkb.png"
+alt="Screenshot of Redeploy Menu"
 layout="responsive"
-width={1512} height={1254} quality={80} />
+width={888} height={493} quality={100} />
```
Comment (ndneighbor) [inline src/pages/deploy/deployments.md:132]: I was under the impression that lowering `quality` would make the image load faster? Am I mistaken?

**Review by ndneighbor — APPROVED** [review]
Tiny nit, but okay to merge after its addressed.

File: src/pages/deploy/deployments.md (hunk 2 above)
Comment (coffee-cup) [reply src/pages/deploy/deployments.md:132]: It should yes. But I realized that this image is already about half the size as the other images so I up'ed the quality to hopefully be comparable

**Pull request merged**